modules/check_in/app/services/v2/chip/service.rb

```diff
@@ -90,6 +98,19 @@ def identifier_params
         }
       end
 
+      def demographic_confirmations
+        demographics_needs_update = check_in_body[:demographics_up_to_date] ? false : true
```

Comment:
no, it specifically has to be reversed since the FE sends us the flag whether the data is up to date or not, and CHIP expects us to send whether the data needs to be updated or not. so if 'up to date' = true, 'update needed' = false